README.md

```diff
@@ -1 +1,57 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+
+> My First Microverse Project.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo 
+
+[Coming Soon]
+
+
+## Getting Started
+
+**Go to your Cli and follow the steps below**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Clone the repository: go to Your Cli and run "git clone <repository url>"
+
+### change into the repository:  run 'cd <repository name>'
+
+### Install all the project dependences: Run 'npm Insttall' in your cli
+
+### now you can view the project in the browser using any web server of choice
+
+
+
+## Author
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/Eugeneiyukhun)
```

Comment:
**[OPTIONAL]:** I would highly recommend adding the rest of your social/work platform handles as this will ensure that anyone who visits your repo will be able to reach out to you by accessing any of these platforms :busts_in_silhouette: :nerd_face: